desktop/mac/permission-requirements.md

```diff
@@ -60,6 +125,10 @@ ERRO[0003] error waiting for container: context canceled
 >
 > The command may fail with the same error if the user takes too long to authorize the prompt to start the helper process, as it may timeout.
 
+<hr>
+</div>
+</div>
+
 ### Ensuring `localhost` and `kubernetes.docker.internal` are defined
```

Comment:
@MihaelaStoica does this need to change in any way for 4.18?

Reply:
Yes, we are not doing this anymore, so perhaps we can say that it's now the user's responsibility to ensure that
`localhost` is resolved to `127.0.0.1` and , if kubernetes is used, that `kubernetes.docker.internal` is resolved to `127.0.0.1`

@aiordache please confirm